src/Integration/NewConnectedMode/IConfigurationPersister.cs

```diff
@@ -95,14 +92,6 @@ private BindingConfiguration CreateBindingConfiguration(string bindingPath, Boun
                     return null;
                 }
             }
-
-            return writeSettings;
-        }
-
-        private struct WriteSettings
-        {
-            public Action<string> OnSuccessfulFileWrite { get; set; }
```

Comment:
The `OnSuccessfulFileWrite` field was not being set, so I removed the `WriteSettings` struct and changed the `GetWriteSettings` method just to return the directory path.